### ISSUE TITLE
warning from xgboost

### ISSUE THREAD

**Issue by mikelty** [EpistasisLab/tpot#1139]
As much as I like running topt, there's a ton of the same warning from xgboost while running.
The warning reads:
[10:00:40] WARNING: C:/Users/Administrator/workspace/xgboost-win64_release_1.3.0/src/learner.cc:1061: Starting in XGBoost 1.3.0, the default evaluation metric used with the objective 'binary:logistic' was changed from 'error' to 'logloss'. Explicitly set eval_metric if you'd like to restore the old behavior.
How to disable this warning? Thanks.

**Comment by weixuanfu** [comment]
Please try this workaround below for using TPOTClassifier with xgboost >=1.30. 

```python
from tpot import TPOTClassifier
from tpot.config import classifier_config_dict
from sklearn.datasets import load_digits
from sklearn.model_selection import train_test_split
import numpy as np
digits = load_digits()
X_train, X_test, y_train, y_test = train_test_split(digits.data, digits.target,
                                                    train_size=0.75, test_size=0.25)
# fit xgboost v1.3.0 API

classifier_config_dict['xgboost.XGBClassifier'] = {
    'n_estimators': [100],
    'max_depth': range(1, 11),
    'learning_rate': [1e-3, 1e-2, 1e-1, 0.5, 1.],
    'subsample': np.arange(0.05, 1.01, 0.05),
    'min_child_weight': range(1, 21),
    'n_jobs': [1], # replace "nthread"
    'verbosity': [0] # add this line to slient warning message
}
        
# for a quick test
tpot = TPOTClassifier(generations=2, population_size=10, verbosity=2,
                      config_dict=classifier_config_dict)
tpot.fit(X_train, y_train)
```

For TPOTRegressor, the config_dict should be:

```python
from tpot.config import regressor_config_dict

regressor_config_dict['xgboost.XGBRegressor'] =
{
        'n_estimators': [100],
        'max_depth': range(1, 11),
        'learning_rate': [1e-3, 1e-2, 1e-1, 0.5, 1.],
        'subsample': np.arange(0.05, 1.01, 0.05),
        'min_child_weight': range(1, 21),
        'objective': ['reg:squarederror'],
        'n_jobs': [1], # replace "nthread"
        'verbosity': [0] # add this line to slient warning message
}
```
We will add a patch about this issue to TPOT next week.

**Comment by weixuanfu** [comment]
I made a patch about this issue in [v0.11.6.post3](https://pypi.org/project/TPOT/0.11.6.post3/). Please check this version.

**Comment by mikelty** [comment]
ok, thanks.

**Comment by Gokulrajkmv** [comment]
Thank you very much!!
